src/cli/commands/dev.ts

```diff
@@ -98,7 +112,7 @@ export class Dev implements Command {
     })
 
     const watcher = await createWatcher({
-      entrypointScript: transpiledStartModule,
+      entrypointScript: args['--reflection'] ? '' : transpiledStartModule,
```

Comment:
We use an empty script when in reflection mode so that no code is running. The watcher will keep running though and so will reflection line 58-66

Reply:
Looks like a good code comment.